index.ejs

```diff
@@ -6,11 +6,9 @@
 <% htmlWebpackPlugin.options.scripts = htmlWebpackPlugin.options.scripts || [] %>
 
 <!DOCTYPE html>
-<!--[if lt IE 7 ]>             <html class="ie6" lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if IE 7 ]>                <html class="ie7" lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if IE 8 ]>                <html class="ie8" lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if IE 9 ]>                <html class="ie9" lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <![endif]-->
-<!--[if (gt IE 9)|!(IE)]><!--> <html lang="en"<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>> <!--<![endif]-->
+<html
+
+<% if (htmlWebpackPlugin.files.lang) { %> lang="<%= htmlWebpackPlugin.options.lang %>"<% } %><% else { %> lang="en"<% } %><% if (htmlWebpackPlugin.files.manifest)<% if (htmlWebpackPlugin.files.manifest) { %> manifest="<%= htmlWebpackPlugin.files.manifest %>"<% } %>>
```

Comment:
@azat-io , I think this default will work, but I wonder if you wouldn't mind using the strategy we've used for defaults thus far instead.  For instance, see line 6:

```
<% htmlWebpackPlugin.options.scripts = htmlWebpackPlugin.options.scripts || [] %>
```

Something like that at the top of the file would be nice.  Then you could avoid the else branch inline.  Sound ok?